src/fsharp/TastOps.fs

```diff
@@ -3004,7 +3004,22 @@ let isByrefLikeTy g m ty =
 
 let isSpanLikeTy g m ty =
     isByrefLikeTy g m ty && 
-    not (isByrefTy g ty) 
+    not (isByrefTy g ty)
+
+let isSpanTyconRef (g: TcGlobals) m (tcref: TyconRef) =
+    if isByrefLikeTyconRef g m tcref then
+        match tcref.ILTyconInfo with
+        | TILObjectReprData(_, _, ilTypeDef) -> ilTypeDef.Name = "System.Span`1"
```

Comment:
You defined that string as literal somewhere below

Reply:
I guess we could use that.